src/Bicep.Core/TypeSystem/BooleanLiteralType.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.Core.TypeSystem;
+
+public class BooleanLiteralType : TypeSymbol
+{
+    public BooleanLiteralType(bool value)
+        : base(value.ToString())
```

Comment:
`.ToString()` capitalizes the first letter of true/false

```suggestion
        : base(value ? "true" : "false")
```